rules/game_process.tex

```diff
@@ -75,7 +81,16 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
-The referee will announce the transition from the \texttt{initial} to \texttt{ready} state by raising both hands over its head. The referee will wear red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following year.}.
+At the scheduled time, the referee will announce that the transition from the \texttt{setup} to \texttt{initial} will be executed 60 seconds later.
+The referee shall call ``Initial in 60 seconds''. 
+The GameController will execute this transition automatically once the time has passed. 
+If all humans from playing teams have left the field, the head referee can announce the transition earlier by calling ``Initial''.
+
+
+The referee should wait a random time between 10 to 120 seconds before announcing the transition to \texttt{ready}. 
```

Comment:
Since the referee must not signal until the initial state is active, I suggest:

"Once the game controller operator has confirmed the initial state, the referee should wait a random time..."